src/components/Select/Select.tsx

```diff
@@ -0,0 +1,17 @@
+interface ISelect {
+  title: string;
+}
+
+const Select = ({ title }: ISelect) => {
+  return (
+    <div className='select'>
+      <div className='select-btn'>{title}</div>
+      <div className='select-contain'>
+        <div className='select-option'>price</div>
+        <div className='select-option'>year</div>
```

Comment:
Why did you hardcoded this value? You could make this component reusable. Also, please take a look at the Compound pattern